### ISSUE TITLE
etserver exits with no error during systemd startup on Fedora 30

### ISSUE THREAD

**Issue by jeffgoh** [MisterTea/EternalTerminal#216]
Both systems are on et-5.1.10 - et-5.1.10-1.fc29.src.rpm vs et-5.1.10-1.fc30.src.rpm

but,

On Fedora 30:
● et.service - Eternal Terminal
   Loaded: loaded (/usr/lib/systemd/system/et.service; enabled; vendor preset: disabled)
   Active: inactive (dead) since Thu 2019-08-08 09:23:01 +08; 2s ago
  Process: 30637 ExecStart=/usr/bin/etserver --daemon --cfgfile=/etc/et.cfg (code=exited, status=0/SUCCESS)
 Main PID: 30638 (code=exited, status=0/SUCCESS)

Aug 08 09:23:01 [redacted] systemd[1]: Starting Eternal Terminal...
Aug 08 09:23:01 [redacted] systemd[1]: Started Eternal Terminal.
Aug 08 09:23:01 [redacted] systemd[1]: et.service: Succeeded.

but works fine on Fedora 29:

● et.service - Eternal Terminal
   Loaded: loaded (/usr/lib/systemd/system/et.service; enabled; vendor preset: disabled)
   Active: active (running) since Tue 2019-07-16 21:40:54 +08; 3 weeks 1 days ago
 Main PID: 1595 (etserver)
    Tasks: 10 (limit: 4915)
   Memory: 8.0M
   CGroup: /system.slice/et.service
           └─1595 /usr/bin/etserver --daemon --cfgfile=/etc/et.cfg

Jul 16 21:40:54 [redacted] systemd[1]: Starting Eternal Terminal...
Jul 16 21:40:54 [redacted] systemd[1]: Started Eternal Terminal.

**Comment by MisterTea** [comment]
Please enable verbose logging by editing /etc/et.cfg and then paste the contents of /tmp/etserver* here

**Comment by MisterTea** [comment]
@jeffgoh This should be fixed in et 6, but you will need to build from source and copy the binaries & new et.service file manually as there isn't a package yet

**Comment by singalen** [comment]
Just installed master from source.

```
Redirecting to /bin/systemctl status et.service
● et.service - Eternal Terminal
   Loaded: loaded (/etc/systemd/system/et.service; disabled; vendor preset: disabled)
   Active: inactive (dead)

Aug 27 13:30:42 mylocalhost systemd[1]: et.service: Succeeded.
Sep 23 14:31:24 mylocalhost systemd[1]: Starting Eternal Terminal...
Sep 23 14:31:24 mylocalhost systemd[1]: Started Eternal Terminal.
Sep 23 14:31:24 mylocalhost systemd[1]: et.service: Succeeded.
Sep 23 14:33:32 mylocalhost systemd[1]: Starting Eternal Terminal...
Sep 23 14:33:32 mylocalhost systemd[1]: Started Eternal Terminal.
Sep 23 14:33:32 mylocalhost systemd[1]: et.service: Succeeded.
Sep 23 14:49:16 mylocalhost systemd[1]: Starting Eternal Terminal...
Sep 23 14:49:16 mylocalhost systemd[1]: Started Eternal Terminal.
Sep 23 14:49:16 mylocalhost systemd[1]: et.service: Succeeded.
```
There's nothing in the system journal about it. `/var/run/etserver.pid` exists, but no process with such pid is running.

When I run it by hand with `sudo /usr/local/bin/etserver --daemon --cfgfile=/etc/et.cfg &`, it seems to fork and runs seemingly fine afterwards, but I cannot connect to it:

```$ et user@mylocalhost.local
Error in authentication with etserver: Exception: Argument ‘--v=0’ starts with a - but has incorrect syntax

Remote shell for the busy and impatient
Usage:
  et [OPTION...] [user@]hostname[:port]```

Same happens when I just run it as `sudo /usr/local/bin/etserver --cfgfile=/etc/et.cfg`.

Setting `verbose = 1` in et.cfg doesn't yield any additional information.

**Comment by zahanm** [comment]
Hm - I'm seeing the same, but on Ubuntu on `et` 6. It was working fine when I first installed it, weirdly. I installed from the `ppa:jgmath2000/et` repository.

```
> systemctl status et
● et.service - Eternal Terminal
   Loaded: loaded (/lib/systemd/system/et.service; enabled; vendor preset: enabled)
   Active: inactive (dead) since Sun 2019-10-13 09:40:45 PDT; 2s ago
  Process: 7078 ExecStart=/usr/bin/etserver --daemon --cfgfile=/etc/et.cfg (code=exited, status=0/SUC
 Main PID: 7087 (code=exited, status=0/SUCCESS)

Oct 13 09:40:45 [redacted] systemd[1]: Starting Eternal Terminal...
Oct 13 09:40:45 [redacted] systemd[1]: Started Eternal Terminal.
```

And these are my versions
```
> et --version
et version 6.0.4
> lsb_release --all
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.3 LTS
Release:	18.04
Codename:	bionic
```

**Comment by zahanm** [comment]
Update: this appears to be fixed in `Version: 6.0.4-bionic2`
